template/provision/templates/application.yml.template

```diff
@@ -0,0 +1,12 @@
+# These environment variables are applied to all environments
+SECRET_KEY_BASE: {{secret_key_base}}
```

Comment:
the wording could definitely be confusing to someone new to this
```# These environment variables are applied to all environments```
kinda makes it sound like it's the same value across all environments